README.rst

```diff
@@ -268,6 +268,17 @@ Example configuration for a single reader::
               "name": "journal.oom_killer"
           }
       ],
+      "secret_filter_metrics": true,
+      "secret_filters": [
+        {
+          "pattern": "SENSITIVE",
+          "redaction_message": "[REDACTED]"
+        }
+        {
+          "pattern": "(.*?\\s?)(SECRET)(\\s?.*)",
```

Comment:
I'd use something else as an example for this because both the leading and trailing pattern match anything and this works exactly the same way as the first example. (Same for the other location that uses this same example pattern.)

Reply:
I've updated the first main example to only touch on the simple example, but in the complex example description down below i'd like to keep an explanation of the second approach.

Reply:
I see what you mean, I updated it with a couple of examples, using simpler syntax and a more reasonable example. 
Also swapped out "redaction_message" with "replacement" too as I like your suggestion more.